### PR TITLE
Swaps out dogstatsd # of packets for total bytes

### PR DESCRIPTION
**Pull request by scottopell**
### What does this PR do?

Updates the set of default agenttelemetry

### CODE REVIEW

File: comp/core/agenttelemetry/impl/config.go [hunk 1]
```diff
@@ -200,13 +200,8 @@ var defaultProfiles = `
...
         - name: transactions.retries
-        - name: dogstatsd.udp_packets
-          aggregate_tags:
-            - state
-        - name: dogstatsd.uds_packets
-          aggregate_tags:
-            - transport
-            - state
+        - name: dogstatsd.udp_packets_bytes
+        - name: dogstatsd.uds_packets_bytes
```
Comment (iglendd) [inline comp/core/agenttelemetry/impl/config.go:204]: This change should be accommodated with the corresponding change on actual metric (at least currently). Second question you are not interested in tag partitioning (less values to emit the better of course).

Comment (iglendd) [reply comp/core/agenttelemetry/impl/config.go:204]: Oh wait a seconds, are there actual metrics like that? Let me check.

Comment (iglendd) [reply comp/core/agenttelemetry/impl/config.go:204]: Ok, never mind my comment

**Review by iglendd — APPROVED** [review]

**Review by carlosroman — APPROVED** [review]

**Pull request merged**